lib/Doctrine/Sequence/Mssql.php

```diff
@@ -61,11 +67,11 @@ public function nextId($seqName, $onDemand = true)
                 // we know the first id = 1 so initialize the
                 // sequence at 2
                 try {
-                    $result = $this->conn->export->createSequence($seqName, 2);
```

Comment:
`createSequence` here doesn't return anything, and this `$result` variable isn't used.